docs/pluggable_storage.rst

```diff
@@ -57,6 +57,9 @@ AzurePinotFS requires the following configs according to your environment:
 
 adl.accountId, adl.authEndpoint, adl.clientId, adl.clientSecret
 
+You can find the parameters in your account as follows:
+https://stackoverflow.com/questions/56349040/what-is-clientid-authtokenendpoint-clientkey-for-accessing-azure-data-lake
```

Comment:
This does not help in showing an example of how the Pinot ADLS config looks like, right?

Reply:
It displays where clients need to look for params, which could be helpful for clients confused about meaning